tlf.1.in

```diff
@@ -2682,10 +2682,6 @@ instead.
 Points for countries in country list.
 .
 .TP
```

Comment:
I would be worth spending an own chapter to scoring. At least to have the basic logic documented.

Reply:
True. Points and multiplier scoring needs to be documented. I was undecided if we do it in man page or should use a separate document. As a start the keywords in man page should be sorted and grouped together.